app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java

```diff
@@ -161,16 +157,11 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     @Override
     public void onRequestPermissionsResult(int requestCode,
                                            @NonNull String[] permissions, @NonNull int[] grantResults) {
-        if (grantResults.length < 1)
-            return;
-        if (requestCode == REQUEST_PERMISSIONS_CODE) {
-            if (grantResults[0] == PackageManager.PERMISSION_GRANTED) {
-                mPermissionGranted = true;
-                selectImages();
-                StringUtils.getInstance().showSnackbar(mActivity, R.string.snackbar_permissions_given);
-            } else
-                StringUtils.getInstance().showSnackbar(mActivity, R.string.snackbar_insufficient_permissions);
-        }
+        PermissionsUtils.getInstance().handleRequestPermissionsResult(mActivity, grantResults,
```

Comment:
You can create 1 CL with this new `PermissionsUtils` files introduced

Reply:
What do you mean with Cl? a commit or a pull request?

Reply:
I mean, Pull request